src/plugins/expressions/public/expression_renderer.test.tsx

```diff
@@ -54,68 +56,49 @@ describe('ExpressionRenderer', () => {
 
     const instance = mount(<ExpressionRendererImplementation expression="" />);
 
-    loadingSubject.next();
+    act(() => {
+      loadingSubject.next();
+    });
+
     instance.update();
 
     expect(instance.find(EuiProgress)).toHaveLength(1);
 
-    renderSubject.next(1);
+    act(() => {
+      renderSubject.next(1);
+    });
 
     instance.update();
 
     expect(instance.find(EuiProgress)).toHaveLength(0);
 
     instance.setProps({ expression: 'something new' });
-    loadingSubject.next();
+    act(() => {
+      loadingSubject.next();
+    });
     instance.update();
 
     expect(instance.find(EuiProgress)).toHaveLength(1);
-
-    renderSubject.next(1);
-    instance.update();
-
-    expect(instance.find(EuiProgress)).toHaveLength(0);
-  });
-
-  it('should display an error message when the expression fails', () => {
```

Comment:
You've changed the behavior this test was testing- why?

Reply:
Default error handling of `expression_renderer.tsx` now fallbacks to default error handling of `error.ts`. This was done primarily to reduce number of code paths to support. 

>   This looks like it's changing the default behavior of the error handler when the expression fails to render. Try going to Lens and configuring only a Y axis.

I guess I indeed missing the change, where I had to explicitly override error handling, where previous default of `expression_renderer.tsx` was expected. 
I guess it should be done here: `x-pack/legacy/plugins/lens/public/editor_frame_plugin/embeddable/expression_wrapper.tsx` but I am struggling to find it on UI. Appreciate help 🙏 

> Try going to Lens and configuring only a Y axis.
I am likely doing it wrong, but I am getting a blank screen for Stacked bar chart in this case in master. Any suggestions?